src/standalone-mocks/useChrome-mock.ts

```
@@ -1,20 +1,12 @@
-export const useChrome = (): Record<string, unknown> => {
+export const useChrome = () => {
   useEffect(() => {
-    if (
-      (window as Record<string, any>).insights &&
-      (window as Record<string, any>).insights.chrome
-    ) {
-      (window as Record<string, any>).insights.chrome.init();
-      (window as Record<string, any>).insights.chrome.identifyApp(
-        'assisted-migration-app',
-      ); // Your appname from package.json
+    if (window.insights && window.insights.chrome) {
```

ESLint

[error] Calling chrome API from the window object is deprecated.

(rulesdir/no-chrome-api-call-from-window)